Setup.sh

```diff
@@ -108,10 +108,10 @@ cp -R "${BINARIES_DIR}"/Headers/include/ "${WORKER_SDK_DIR}"
 
 if [[ -d "${SPATIAL_DIR}" ]]; then
     echo "Copying standard library schemas to ${SCHEMA_STD_COPY_DIR}"
-    cp -R "${BINARIES_DIR}/Programs/schema" "${SCHEMA_STD_COPY_DIR}"
+    cp -R "${BINARIES_DIR}/Programs/schema/." "${SCHEMA_STD_COPY_DIR}"
```

Comment:
why is this one `.` and the one below `*` ?

Reply:
probably should actually be a `.` for both... I noticed that `*` caused problems when you have subfolders...

Reply:
Hmmm, what's the difference? Does * not take into account subfolders?

Reply:
yeah, it doesn't seem to take them into account...